reverseCase/reverseCase.js

```diff
@@ -4,4 +4,16 @@
  * Assume that each string will contain only spaces and letters.
  */
 
+const reverseCase = (str) => {
+  starr = str.split('');
+  for (let i = 0; i < starr.length; i++) {
+    if (starr[i].charCodeAt(0) >= 65 && starr[i].charCodeAt(0) <= 90) {
```

Comment:
Hey that's awesome using charCode. I wouldn't have ever thought to attempt to solve this that method! Nice work. Your code is well organized and logic easy to follow